front/app/modules/commercial/insights/utils/getInputsCategoryFilter.ts

```diff
@@ -0,0 +1,13 @@
+const getSelectedCategoryFilter = (
```

Comment:
@SaraBCL Just to follow up on your comment from my previous PR, there is no easy way to convert this into a custom hook which would access the query parameters by itself. The reason is that the version of React Router we are using only allows us to use HOCs (`withRouter`) to do this and it only works with components (not hooks). Let's leave it as is for now and if we get to a point where we manage to update the router, we can revisit it.

Unless you have a different idea, in which case I am open to suggestions :)

Reply:
Thanks for looking into it ! I don't think we'll change version in any foreseeable future, but we'll try to keep these small things in mind when considering again. We could also probably find another more hacky way but the payoff doesn't seem high enough for adding some technical debt just yet.

Reply:
Agreed 👍